krausening/src/test/java/org/technologybrewery/krausening/TestEncryptedKrausening.java

```diff
@@ -28,6 +29,13 @@ public void testEncryptedPropertyReadProvidesDecryptedValue() {
 		krausening.loadProperties();
 		Properties properties = krausening.getProperties(ENCRYPTED_PROPERTIES);
 		assertNotNull(properties);
```

Comment:
I: the lamba breaks things with the encryptor (and arguably shouldn't be in this imperative codebase anyway)